kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -140,6 +140,21 @@ paths:
           required: false
           schema:
             type: integer
+        - name: showInternal
+          in: query
+          required: false
```

Comment:
Shouldn't we set default values for these optional parameters?

Reply:
I intentionally left them as optionals with no defaults
No value is the default value

Moreover using `default` doesn't change the generated code (except for annotations) both for be and fe

Reply:
Ok
Let's keep it as is